src/connection/passwordless/actions.js

```diff
@@ -28,15 +28,25 @@ function getErrorMessage(m, error) {
     key = 'bad.phone_number';
   }
 
+  if (error.code === 'invalid_captcha') {
+    const captchaConfig = l.passwordlessCaptcha(m);
+    key = captchaConfig.get('provider') === 'recaptcha_v2' ? 'invalid_recaptcha' : 'invalid_captcha';
```

Comment:
Do we need to think about recaptcha enterprise here too?

Reply:
Thank you for the callout. Since `recaptcha_v2` and `recaptcha_enterprise` both offer the same UX, I've made the error messages match everywhere we are showing captcha. 👍